src/EnergyPlus/ChillerAbsorption.hh

```diff
@@ -223,15 +197,12 @@ namespace ChillerAbsorption {
 
 struct ChillerAbsorberData : BaseGlobalStruct
 {
-    int numAbsorbers = 0;
     bool getInput = true;
     Array1D<ChillerAbsorption::BLASTAbsorberSpecs> absorptionChillers;
 
     void clear_state() override
     {
-        this->numAbsorbers = 0;
-        this->getInput = true;
-        this->absorptionChillers.deallocate();
+        *this = ChillerAbsorberData();
```

Comment:
These *Data classes are starting to look a lot alike.  I think that's a good sign that things are getting normalized and will be opening up some refactoring next-steps sooner rather than later.